sunbeamlib/initialize.py

```diff
@@ -42,8 +45,15 @@ def main():
     else:
         args.conda_fp = args.conda_fp.absolute()
 
-    config = create_blank_config(
-        args.conda_fp, args.project_fp, template=args.server)
-    
+    if args.custom_configfile_fp is None:
+        config = create_blank_config(
+            args.conda_fp, args.project_fp, template=args.server)
+    elif os.path.isfile(args.custom_configfile_fp.name):
```

Comment:
I remove the redundant code. We do need to read the whole file from the open file handle, for the stdout.write in the next step.